quickwit-metastore/src/postgresql/schema.rs

```diff
@@ -30,7 +30,8 @@ table! {
         split_state -> Varchar,
         time_range_start -> Nullable<Int8>,
         time_range_end -> Nullable<Int8>,
-        update_timestamp -> Int8,
+        created_at -> Nullable<Timestamp>,
```

Comment:
Here is my answer. Why are those timestamps nullable? Backward-compatibility?

Reply:
This is to allow Diesel do it's magic, by passing None during create/update, diesel does the auto update. otherwise the value you pass will always take precedence.